config/clients/go/template/client/client.mustache

```diff
@@ -102,7 +104,7 @@ func NewSdkClient(cfg *ClientConfiguration) (*{{appShortName}}Client, error) {
 
 	return &{{appShortName}}Client{
 		config:    clientConfig,
-		APIClient: *apiClient,
+		APIClient: apiClient,
```

Comment:
Why are we changing this?

Reply:
The idea was to prevent their being multiple client objects when setting the headers but as this has all been reverted with the latest update I'll mark this as resolved.